docs/detections/rules-ui-create.asciidoc

```diff
@@ -21,30 +21,42 @@ running, the rule will:
 ** Run and create alerts if existing anomaly scores above the defined threshold
 are discovered.
 ** Issue an error stating the {ml} job was not running when the rule executed.
-* *Threshold rules*: Searches the defined indices and creates a detections alert
+* <<create-threshold-rule, *Threshold rules*>>: Searches the defined indices and creates a detections alert
 when the number of times the specified field's value meets the threshold during
 a single execution. When multiple values meet the threshold, an alert is
 generated for each value.
 +
 For example, if the threshold `field` is `source.ip` and its `value` is `10`, an
 alert is generated for every source IP address that appears in at least 10 of
 the rule's search results.
+* <<create-eql-rule, *Event correlation*>>: Searches the defined indices and creates an alert when results match an
+{ref}/eql.html[Event Query Language (EQL)] query.
+* <<create-indicator-rule, *Indicator match*>>: Creates an alert when {es-sec} index field values match
+field values defined in the specified threat indices. For example, you
```

Comment:
I noticed that there is mention of `threat indices` throughout this doc but the UI calls them `indicator index patterns`. I know these changes were last minute and you all have already made updates related to indicator match rules, but It might be better to have the docs match the terminology in the UI in this case also.

![image](https://user-images.githubusercontent.com/616158/98721224-b70e5580-235e-11eb-8bf8-947dce587f15.png)